Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["libraries/*"]
+members = ["libraries/*", "udp-spammer"]
```

Comment:
the `udp-spammer` tool should probably be put in a `utilities/` directory. This would also let us just say `utilities/*` in Cargo.toml